content/en/docs/setup/install/operator/index.md

```diff
@@ -73,9 +73,15 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-## Features and components
+## Customize Istio settings using the IstioControlPlane API
 
-The Operator groups the Istio control plane components by feature, as follows:
+You can change a feature or component setting by using the [IstioControlPlane API](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto).
+
+### Identify the feature or component settings
+
+The Operator groups the Istio control plane components by feature, which determines the required flags to use with `manifest apply` command.
```

Comment:
Line above is not quite right (the flags are not determined by the API structure). I suggest the following text right after the link to IstioControlProto API:
"The API groups Istio control plane components by feature, as shown in the table below:"

Reply:
fixed